UnitsNet.Tests/CustomCode/TorqueTests.cs

```diff
@@ -52,6 +52,12 @@ public class TorqueTests : TorqueTestsBase
 
         protected override double PoundalFeetInOneNewtonMeter => 23.730360404231938;
 
+        protected override double GramForceCentimetersInOneNewtonMeter => KilogramForceCentimetersInOneNewtonMeter / 1000;
```

Comment:
Test case values should be constants, preferably looked up in some reliable source like google, wiki or online converter app. The idea is to double check the functions in the JSON.